delete.go

```diff
@@ -39,7 +42,41 @@ EXAMPLE:
 		},
 	},
 	Action: func(context *cli.Context) error {
-		// TODO
-		return nil
+		return delete(context.String("container-id"), context.Bool("force"))
 	},
 }
+
+func delete(containerID string, force bool) error {
+	// Checks the MUST and MUST NOT from OCI runtime specification
+	if err := checkDeleteParamsIntegrity(containerID); err != nil {
+		return err
+	}
+
+	if _, err := vc.StopPod(containerID); err != nil {
+		return err
+	}
+
```

Comment:
I agree with the idea, but we cannot do that. In order to do that, you would need to get the status switched from running to stopped inside virtcontainers, but in case we are calling KillContainer(), we don't update the pod/container state because we don't know what the signal did to the process. And because we rely on virtcontainers to get the state, there is no way a kill from OCI runtime will switch the state to stopped. Then if we don't force, we will always get an error...